db/seeds.rb

```diff
@@ -5,3 +5,5 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+Admin.delete_all
+Admin.create!({email: "admin@receiptyourself.com", password: "receiptyourself", password_confirmation: "receiptyourself" })
```

Comment:
Have you tried logging in with these credentials? I remember I had to skip email confirmation in this file so I could login.